src/test/skript/tests/syntaxes/expressions/ExprDuplicateCooldown.sk

```diff
@@ -0,0 +1,25 @@
+# includes: ExprDuplicateCooldown, EffAllayDuplicate, EffAllayCanDuplicate, CondAllayCanDuplicate
+test "duplicate cooldown":
+	spawn an allay at test-location:
+		set {_entity} to entity
+
+	set the duplication cooldown of {_entity} to 5 seconds
+	assert the duplication cooldown of {_entity} is 5 seconds with "Duplication cooldown was not set"
+	add 3 seconds to the duplication cooldown of {_entity}
+	assert the duplication cooldown of {_entity} is 8 seconds with "Duplication cooldown did not have 3 seconds added"
+	remove 5 seconds from the duplication cooldown of {_entity}
+	assert the duplication cooldown of {_entity} is 3 seconds with "Duplication cooldown did not have 5 seconds removed"
+	clear the duplication cooldown of {_entity}
+	assert the duplication cooldown of {_entity} is 0 seconds with "Duplication cooldown time was not cleared"
```

Comment:
Needs to test removing more seconds that the cooldown currently is, as well as setting to a negative value, and adding a negative amount of seconds that would make the cooldown negative.